schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl

```diff
@@ -358,7 +376,7 @@ protocol CommonParticipant {
         /**
         Clinical Indication
         */
-        union {null, string} clinicalIndication;
+        union {null, ClinicalIndication} clinicalIndication;
```

Comment:
do we want clinical indication at all in tumour sample?

Reply:
@priesgo  we have been reviewing with the Clare Craig and Alona from the Cancer team, and I have reviewed with how TOMs will input data for the clinicalIndication. It should be at referral level, as there is only one clinicalIndication for a whole referral, despite having many tumour samples.  But for future proofing, we can put clinical indication at referral Test level. FYI @Antonior26

Reply:
@priesgo I will remove from the tumour sample.  That was agreed yesterday with Clare Craig, Alona, and Antonio